src/components/divided-view/divided-view.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, ReactNode } from 'react'
+import { StyleSheet, View, TouchableWithoutFeedback } from 'react-native'
+import Svg, { Path } from 'react-native-svg'
+import { paperTheme } from '../../theme/paper-theme'
+import { Center } from '../center';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+const styles = StyleSheet.create({
+    container: {
+        display: 'flex',
+        height: '100%',
+        width: '100%',
+    },
+    background: {
+        flex: 1,
+        alignItems: 'center'
+    },
+    childrenCont: {
+        height: '50%',
+        padding: '5%',
+    },
+    upper: {
+        position: 'absolute',
+    }
+});
```

Comment:
I would like to change this to:
```suggestion
const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: 'center',
        alignContent: 'stretch',
    },
    background: {
        position: 'absolute',
        height: '100%',
        width: '100%',
    },
    childrenCont: {
        flex: 1,
        padding: 25
    },
});
```